src/pybind/ceph_volume_client.py

```diff
@@ -377,6 +380,8 @@ def _recover_auth_meta(self, auth_id, auth_meta):
                 # VMeta update looks clean. Ceph auth update must have been
                 # clean.
                 if vol_meta['auths'][auth_id] == want_auth:
+                    auth_meta['subvolumes'][volume]['dirty'] = False
```

Comment:
Do we need this line?

Reply:
Yes, right? If vol-metadata file is not dirty, auth-metadata needs to be marked non-dirty?

Reply:
Yes, it was an existing bug, I fixed in mgr/volumes while testing. I will add it as separate commit.

Reply:
Done